clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/spring/client/properties/CamundaClientCloudProperties.java

```diff
@@ -45,14 +48,25 @@ public void setClusterId(final String clusterId) {
     this.clusterId = clusterId;
   }
 
+  @Deprecated
+  @DeprecatedConfigurationProperty(replacement = "camunda.client.cloud.domain")
```

Comment:
For me as a learning, this annotation has no functional impact but only instructs spring to highlight a deprecated prop is used? (so is informational)

In `CamundaClientLegacyPropertiesMappingsLoader` we have the actual functional handling based on the `camunda-client-legacy-property-mappings.properties` content, that will load the deprecated value if the replacement is not present, right?